src/plot_api/plot_api.js

```diff
@@ -196,8 +196,8 @@ Plotly.plot = function(gd, data, layout, config) {
 
         Registry.getComponentMethod('legend', 'draw')(gd);
         Registry.getComponentMethod('rangeselector', 'draw')(gd);
-        Registry.getComponentMethod('updatemenus', 'draw')(gd);
         Registry.getComponentMethod('sliders', 'draw')(gd);
+        Registry.getComponentMethod('updatemenus', 'draw')(gd);
```

Comment:
> Not necessary now but might be a better solution if more cases arise.

Yeah. Maybe the **best** solution would have the dropdown buttons in a separate layer above `<g class="infolayer">`. 

My solution aims to fix the 99% cases for users ASAP.